lib/echo/echo_client.rb

```diff
@@ -133,7 +147,7 @@ def create_order(granule_query, option_id, option_name, option_model, user_id, t
       Rails.logger.info "User info response: #{user_info_response.body.inspect}"
       Rails.logger.info "Submission response: #{submission_response.body.inspect}"
 
-      {order_id: id, response: submission_response, count: granules.size}
+      {order_id: id, response: submission_response, count: granules.size, dropped_granules: dropped_granules}
```

Comment:
dropped_granules contains all of the granule metadata, which could be a lot of information, way more than we need. Just store enough to display and re-query. id and name (where name is producer_granule_id or title)